safe/gui/tools/peta_jakarta_dialog.py

```diff
@@ -196,7 +197,7 @@ def accept(self):
         QtGui.qApp.setOverrideCursor(QtGui.QCursor(QtCore.Qt.WaitCursor))
 
         source = (
-            'https://data.petabencana.id/floods'
+            'https://data-dev.petabencana.id/floods'
```

Comment:
ups you need to change this. We don't use data-dev.petabencana.id, we use dev.petabencana.id.